package.json

```
@@ -41,21 +43,16 @@
     "generate-pending-release-diffs": "bin/generate-pending-release-diffs.sh",
     "format-js": "wp-scripts format",
     "lint-js": "wp-scripts lint-js",
-    "format-php": "composer format",
     "phpstan": "composer phpstan",
-    "lint-php": "composer lint:all",
-    "test-php": "wp-env run tests-cli --env-cwd=/var/www/html/wp-content/plugins/performance composer test:plugins",
-    "test-php-multisite": "wp-env run tests-cli --env-cwd=/var/www/html/wp-content/plugins/performance composer test-multisite:plugins",
+    "lint:php": "composer lint",
+    "lint:php:fix": "composer format",
+    "lint:php:plugins": "cross-env WPP_PLUGIN=$npm_config_plugin php -f bin/phpcs/lint.php",
+    "lint:php:plugins:fix": "cross-env WPP_FIX=1 WPP_PLUGIN=$npm_config_plugin php -f bin/phpcs/lint.php",
```

I guess these need to be updated?

```suggestion
    "lint:php:plugins": "cross-env WPP_PLUGIN=$npm_config_plugin php -f tools/phpcs/lint.php",
    "lint:php:plugins:fix": "cross-env WPP_FIX=1 WPP_PLUGIN=$npm_config_plugin php -f tools/phpcs/lint.php",
```